pkgs/applications/networking/instant-messengers/pidgin/default.nix

```diff
@@ -54,6 +62,8 @@ let unwrapped = stdenv.mkDerivation rec {
     "--disable-meanwhile"
     "--disable-nm"
     "--disable-tcl"
+    "--disable-gtkspell"
+    "--disable-vv"
```

Comment:
The combination of always disabling gtkspell with sometimes passing it is weird, but it's possible that overall the flags need revision.

Reply:
Good catch, these need to be parameterized based on environment. I'll update this patch.